main.go

```diff
@@ -104,12 +104,10 @@ func main() {
 	}
 
 	if err = (&controller.BackstageReconciler{
-		Client:         mgr.GetClient(),
-		Scheme:         mgr.GetScheme(),
-		OwnsRuntime:    ownRuntime,
-		IsOpenShift:    isOpenShift,
-		PsqlImage:      os.Getenv(backstageiov1alpha1.EnvPostGresImage),
-		BackstageImage: os.Getenv(backstageiov1alpha1.EnvBackstageImage),
```

Comment:
I guess you can also remove both fields from the `BackstageReconciler` struct?

Reply:
ah, sure, I though I did